tokeninput.js

```diff
@@ -626,6 +626,7 @@
 
         this.completions = completions;
         if ( !completions.length ) {
+            this.dispatchEvent( 'noCompletions' );
```

Comment:
perhaps this function should dispatch a 'displayedCompletions' event with the completions as a parameter regardless of the number of completions